src/main/java/com/spartronics4915/frc2019/subsystems/CargoIntake.java

```diff
@@ -19,23 +25,35 @@ public static CargoIntake getInstance()
 
     public enum WantedState
     {
-        TODO
+        HOLD, ARM_DOWN, ARM_UP, INTAKE, EJECT, CLIMB
     }
 
     private enum SystemState
     {
-        TODOING
+        HOLDING, ARM_DOWNING, ARM_UPING, INTAKING, EJECTING, CLIMBING
```

Comment:
You will need to add `ARM_DOWNING_FOR_CLIMB` because the arm has to move _all_ the way down for climb.